app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container w-full shadow-lg  p-5">
+  <h1 class="text-4xl text-center">Public Recipes</h1>
+  <ul class="w-full">
+    <% @public_recipes.each do |recipe|%>
+      <%= link_to recipe_path(recipe) do%>
+        <li class ="flex px-5  border shadow-xl m-2 w-full">
+          <div class="grid gap-3 w-1/2 my-2">
+              <h3 class='text-3xl'><strong><%= recipe.name%></strong></h3>
+              <span>By: <%= recipe.user.name%></span>
+          </div><%=  %>
+          <div class="grid gap-3 w-1/2 my-2">
+            <span class="text-2xl">Total food items: <%= recipe.foods.count %></span>
+            <span class="text-2xl">Total price: $<%= recipe.foods.sum(:price) %></span>
+          </div>
+        </li>
+      <% end %><%=  %>
+    <% end %>
+  </ul>
+</div>
```

Comment:
- [ ] [REQUIRED] Let's adjust the design of the public recipe page listing all publicly available recipes from other users with their names and total prices so that it matches with the given wireframe as stated by the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="1163" alt="Screenshot 2023-09-16 at 23 42 37" src="https://github.com/techmoves/recipe-app/assets/58692590/0eac8577-689d-4074-a356-5614544b95c5"> | <img width="943" alt="Screenshot 2023-09-16 at 23 01 42" src="https://github.com/techmoves/recipe-app/assets/58692590/e8a0b9a6-9aea-475c-ac0f-98a060e217dd">